src/operon-runtime/runtime.ts

```diff
@@ -34,14 +35,24 @@ export class OperonRuntime {
     }
 
     const classes: object[] = [];
+    var initFunction = null;
     for (const key in exports) {
+      this.operon.logger.info("found key " + key);
       if (isObject(exports[key])) {
         classes.push(exports[key] as object);
         this.operon.logger.debug(`Loaded class: ${key}`);
       }
+      if (key === "initializeApp") {
```

Comment:
This seems like something that belongs in the operon config file rather than using a magic function name

Reply:
I saw you PR for entry point. Is that CLI only or does it do config file as well ?

Reply:
config file and cli arg

Reply:
Disagree that things needed for code correctness ought to be in the config file.  The should be in the code.

Reply:
We did talk about a decorator approach that would make it a magic decorator name instead of a magic function name... this may be better long term.